crates/dies-world/src/filter/filter_builder.rs

```diff
@@ -0,0 +1,46 @@
+use nalgebra as na;
+use na::{ OMatrix, U2,};
+use na::OVector;
+use na::U4;
+use crate::filter::kalman::Kalman;
+use crate::filter::matrix_gen::{ULMotionModel, WhiteNoise1stOrder};
+
+pub(crate) struct KalmanBuilder {
+    init_std: f64,
+    measurement_std: f64,
+    unit_transition_std: f64
+}
+
+impl KalmanBuilder {
+    pub fn new(init_std: f64, measurement_std: f64, unit_transition_std: f64) -> Self {
+        KalmanBuilder {
+            init_std,
+            measurement_std,
+            unit_transition_std
+        }
+    }
+    #[allow(non_snake_case)]
```

Comment:
I think it would be better to stick to snake_case for the whole project as that is the Rust convention.